src/axom/core/Array.hpp

```diff
@@ -730,22 +730,22 @@ class Array : public ArrayBase<T, DIM, Array<T, DIM, SPACE>>
                   "Cannot call Array<T>::resize() when T is non-trivially-"
                   "constructible. Use Array<T>::reserve() and emplace_back()"
                   "instead.");
-    const StackArray<IndexType, DIM> dims {static_cast<IndexType>(args)...};
+    const StackArray<IndexType, DIM> dims {{static_cast<IndexType>(args)...}};
```

Comment:
👍

Reply:
Is there a reason we don't apply the args with the integral type that is given?

Reply:
I assume this allows users to supply signed/unsigned integral input and convert it to the represented `axom::IndexType` (and avoid warnings/compilation errors due to the templated types)

@publixsubfan might have some additional context.